packages/cloud-cognitive/src/components/Datagrid/styles/useColumnCenterAlign.scss

```diff
@@ -0,0 +1,23 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2020
+* US Government Users Restricted Rights - Use, duplication or disclosure
+* restricted by GSA ADP Schedule Contract with IBM Corp.
+*/
+
+@import './variables';
+
+.#{$block-class}__center-align-header {
+  width: 100%;
+  text-align: center;
+}
+
+.#{$block-class}__center-align-header button {
```

Comment:
If possible, I would use a class to target the button element instead of just applying this style to all buttons inside of the `.#{$block-class}__center-align-header` class

Reply:
It seems like I don't need this line. So, I'll just remove it from the stylesheet.